Does not compile anymore on Mac

Since https://github.com/Mic92/sops-nix/commit/339a5594021a3c1bd1fb3adfd967820eceb71a95 does it not compile anymore on MacOS.

```
       > Building subPackage ./pkgs/sops-install-secrets
       > # github.com/Mic92/sops-nix/pkgs/sops-install-secrets
       > pkgs/sops-install-secrets/main.go:937:82: too many arguments in call to MountSecretFs
       >      have (string, int, bool, bool)
       >         want (string, int, bool)
```

The fourth argument is missing in the darwin function 😅

--- a/pkgs/sops-install-secrets/darwin.go
+++ b/pkgs/sops-install-secrets/darwin.go
@@ -53,7 +53,7 @@ func SecureSymlinkChown(symlinkToCheck string, expectedTarget string, owner, gro
 // mydev=`hdiutil attach -nomount ram://$NUMSECTORS`
 // newfs_hfs $mydev
 // mount -t hfs $mydev /tmp/mymount
-func MountSecretFs(mountpoint string, keysGid int, useTmpfs bool, userMode bool) error {
+func MountSecretFs(mountpoint string, keysGid int, _useTmpfs bool, userMode bool) error {
 	if err := os.MkdirAll(mountpoint, 0751); err != nil {
 		return fmt.Errorf("Cannot create directory '%s': %w", mountpoint, err)
 	}

--- a/pkgs/sops-install-secrets/darwin.go
+++ b/pkgs/sops-install-secrets/darwin.go
@@ -53,7 +53,7 @@ func SecureSymlinkChown(symlinkToCheck string, expectedTarget string, owner, gro
 // mydev=`hdiutil attach -nomount ram://$NUMSECTORS`
 // newfs_hfs $mydev
 // mount -t hfs $mydev /tmp/mymount
-func MountSecretFs(mountpoint string, keysGid int, userMode bool) error {
+func MountSecretFs(mountpoint string, keysGid int, useTmpfs bool, userMode bool) error {
 	if err := os.MkdirAll(mountpoint, 0751); err != nil {
 		return fmt.Errorf("Cannot create directory '%s': %w", mountpoint, err)
 	}

